build_runner/lib/src/server/server.dart

```diff
@@ -428,11 +605,20 @@ String _renderPerformance(BuildPerformance performance, bool hideSkipped) {
         display: inline-block;
         margin: 0.5em;
       }
+      .perf-tooltip {
+        margin: 0.5em;
+      }
       </style>
     </head>
     <body>
       <div class="controls-header">
         <p>$showSkippedLink</p>
+        <p>$showSlicesLink</p>
+        <p><b>Sort by</b>: </p>
+        <p>Start Time (<a href="$sortStartTimeAsc">Asc</a>, <a href="$sortStartTimeDesc">Desc</a>)</p>
```

Comment:
I think a dropdown might be nicer, its a little bit more code to handle the selection but makes it more obvious these are mutually exclusive and uses less space in the UI